src/code/game.c

```diff
@@ -392,20 +414,30 @@ void GameState_Init(GameState* gameState, GameStateFunc init, GraphicsContext* g
     gameState->destroy = NULL;
     gameState->running = 1;
     startTime = osGetTime();
+
+#ifdef OOT_DEBUG
     gameState->size = 0;
     gameState->init = NULL;
-    endTime = osGetTime();
+#else
+    gameState->init = NULL;
+    gameState->size = 0;
+#endif
 
-    // "game_set_next_game_null processing time %d us"
-    PRINTF("game_set_next_game_null 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
-    startTime = endTime;
-    GameAlloc_Init(&gameState->alloc);
+    {
+        s32 requiredScopeTemp;
+        endTime = osGetTime();
 
-    endTime = osGetTime();
-    // "gamealloc_init processing time %d us"
-    PRINTF("gamealloc_init 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
+        // "game_set_next_game_null processing time %d us"
+        PRINTF("game_set_next_game_null 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
+        startTime = endTime;
+        GameAlloc_Init(&gameState->alloc);
+
+        endTime = osGetTime();
+        // "gamealloc_init processing time %d us"
+        PRINTF("gamealloc_init 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
+        startTime = endTime;
+    }
 
-    startTime = endTime;
     GameState_InitArena(gameState, 0x100000);
```

Comment:
```c
    {
        s32 requiredScopeTemp;
        endTime = osGetTime();

        // "game_set_next_game_null processing time %d us"
        PRINTF("game_set_next_game_null 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
        startTime = endTime;
        GameAlloc_Init(&gameState->alloc);
    }

    endTime = osGetTime();

    // "gamealloc_init processing time %d us"
    PRINTF("gamealloc_init 処理時間 %d us\n", OS_CYCLES_TO_USEC(endTime - startTime));
    startTime = endTime;
    GameState_InitArena(gameState, 0x100000);
```

Maybe you can group this block with the function it's associated with, instead of putting it in the scope above it

Reply:
yea this is better imo